src/main/java/org/openstreetmap/atlas/tags/annotations/extraction/SpeedExtractor.java

```diff
@@ -18,33 +23,40 @@
     private static final SpeedValidator VALIDATOR = new SpeedValidator();
     private static final String SINGLE_SPACE = " ";
     private static final Logger logger = LoggerFactory.getLogger(SpeedExtractor.class);
+    private static final JsonObject IMPLICIT_SPEED_MAP = new Gson().fromJson(
+            new JsonReader(
+                    new InputStreamReader(SpeedExtractor.class.getClassLoader().getResourceAsStream(
+                            "org/openstreetmap/atlas/tags/annotations/implicit-speed-values.json"))),
```

Comment:
You should be able to pull the resource by..
```java
SpeedExtractor.class.getResourceAsStream("implicit-speed-values.json")
```

Reply:
Because the resource is being shared between a validator and extractor, I placed it in their parent `annotations` package. Thus I have to use the full resource path.